components/reusable-components/navigation/Navigation.tsx

```diff
@@ -7,84 +7,56 @@ import React from 'react';
 import Link from 'next/link';
 import Image from 'next/image';
 import styles from './navigation.module.scss';
-import Button from '../button/Button';
-import solImage from '../../../public/theme/sol.png';
-import moonImage from '../../../public/theme/lune.png';
 import { useTheme } from '../../../app/context/themeContext';
+import solImage from '../../../public/theme/sun.png';
+import moonImage from '../../../public/theme/moon.png';
 
 const Navigation = () => {
   const { theme, toggleTheme } = useTheme();
-  const isLightTheme = theme === 'light';
+  const isSun = theme === 'light';
```

Comment:
I am not quite sure about the naming here, can we try something more descriptive?